nupicaudio/encoders/sound_encoders_test.py

```diff
@@ -0,0 +1,13 @@
+# This file will demanstrate use of available sound encoders:
+
+# WAV
+data="../examples/data/a0007.wav"
+#from nupicaudio.encoders.wave import WAVEncoder #FIXME move to nupicaudio.examples
+from wave import WAVEncoder
```

Comment:
use the absolute import `from nupicaudio.encoders.wave` once `pip installable` #24